ui-v2/app/templates/dc/nodes/-services.hbs

```diff
@@ -16,7 +16,7 @@
         {{#block-slot 'row'}}
             <td data-test-service-name="{{item.Service}}">
               <a href={{href-to 'dc.services.show' item.Service}}>
-                <span data-test-external-source="{{service/external-source item}}" style="background-image: {{css-var (concat '--' (service/external-source item) '-color-svg') 'none'}}"></span>
+                <span data-test-external-source="{{service/external-source item}}" style={{{ concat 'background-image: ' (css-var (concat '--' (service/external-source item) '-color-svg') 'none')}}}></span>
```

Comment:
Does this not give you warnings about unsafe style injections?

Reply:
It did, but it doesn't now after this change :)

Reply:
Interesting. TIL.